### PR TITLE
Rename stork.com to stork in stork rules CRD

### PR DESCRIPTION
**Pull request by harsh-px**

### CODE REVIEW

File: pkg/apis/stork/register.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package stork
 
 const (
 	// GroupName is the group name of the CRD
-	GroupName = "stork.com"
```
Comment (disrani-px) [inline pkg/apis/stork/register.go]: GroupName should be in domain format, stork.libopenstorage.com

Comment (harsh-px) [reply pkg/apis/stork/register.go]: Updated. It's a ghost link and has only symbolic purpose.

Comment (disrani-px) [reply pkg/apis/stork/register.go]: It should actually be used to register the CRD. The actual group name used to generate the boilerplate code in doc.go should also be stork.libopenstorage.com

Comment (harsh-px) [reply pkg/apis/stork/register.go]: fixed the group name

**Pull request merged**